src/cli/cli.cpp

```diff
@@ -7800,6 +7800,37 @@ template <> otError Interpreter::Process<Cmd("trel")>(Arg aArgs[])
             }
         }
     }
+    /**
+     * @cli trel counters
+     * @code
+     * trel counters
+     * Outbound: Packets 4 Bytes 320 Failures 1
+     * Inbound:  Packets 32 Bytes 4000
```

Comment:
not match below `OutputTrelCounters` where `Outbound` is after `Inbound`

Reply:
Done